examples/deepspeed/trainer/trainer.py

```diff
@@ -0,0 +1,55 @@
+import argparse
+import logging
+import yaml
+
+from attrdict import AttrDict
+
+import determined as det
+from determined import pytorch
+from determined.pytorch import deepspeed as det_ds
+
+from model_def import DCGANTrial
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+
+    # Optional arguments for the launch helper
+    parser.add_argument("--local",
```

Comment:
don't think this is needed. like, if you're running local, you'd just do `python3 trainer.py`. if you're on cluster, you'd call the same thing but from the config entrypoint.

in the code, use `local = det.get_cluster_info() is None`